### PR TITLE
travis: add full build to travis

### PR DESCRIPTION
**Pull request by theishshah**

### CODE REVIEW

**Comment by etiennecoutaud** [comment]
Cool enhancement 👍 , instead of push the docker image in a registry you could:
* `$ eval $(minikube docker-env)` before`operator-skd build ...`
* `$ sed -e s/imagePullPolicy: Always/imagePullPolicy: Never/g deploy/operator.yaml`
* `kubectl apply -f deploy/operator.yaml`

It should work, this is a workaround related to #186 

HTH,

**Comment by theishshah** [comment]
Should be in good shape now, travis will build the memcached operator and deploy it onto a minikube cluster

**Comment by rithujohn191** [comment]
It would be great if you could add a small doc for this describing what the tests do and which version of kubernetes it runs

File: .travis.yml [hunk 1]
```diff
@@ -1,15 +1,52 @@
...
+- mv tmp.txt pkg/apis/cache/v1alpha1/types.go
+- echo 'type MemcachedSpec struct {	Size int32 `json:"size"`}' >> pkg/apis/cache/v1alpha1/types.go
+- echo 'type MemcachedStatus struct {Nodes []string `json:"nodes"`}' >> pkg/apis/cache/v1alpha1/types.go
+- operator-sdk generate k8s
+- operator-sdk build quay.io/example/memcached-operator:v0.0.1
+- sed -e 's/imagePullPolicy. Always/imagePullPolicy. Never/g' deploy/operator.yaml
+- kubectl create -f deploy/rbac.yaml
+- kubectl create -f deploy/crd.yaml
+- kubectl create -f deploy/operator.yaml
+- kubectl get deployment
```
Comment (rithujohn191) [inline .travis.yml]: Could you please add some echo statements to log the progress of the test.

File: doc/testing/travis-build.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+## Tool Versions
+* Kubernetes: 1.9.0
+* Minikube: 0.25.2
+* Go: 1.10.1
+
+## Test Workflow
+1. Run unit tests
+2. Ensure proper formatting
+3. Install the sdk
+4. Create memcached operator
```
Comment (rithujohn191) [inline doc/testing/travis-build.md]: Please add a link here to https://github.com/operator-framework/operator-sdk/blob/master/doc/user-guide.md#build-and-run-the-operator

**Pull request merged**